tailwindcss-uswds/tokens/components.json

```diff
@@ -6320,9 +6311,7 @@
     ".usa-breadcrumb__list-item": {
         "whiteSpace": "nowrap",
         "display": "inline",
-        "position": "absolute",
-        "left": "-999em",
-        "right": "auto"
+        "position": "relative"
```

Comment:
This is my first day using tailwind so I would like a second opinion from Mark, but I have a feeling we shouldn't be changing classnames directly, instead adding custom classnames when needed. Thoughts?

Reply:
Very good point; I reverted back to the original styling for that class in `components.json` and overrode the two Breadcrumb components' styling as so: 

`<Breadcrumb className="relative left-0">`

I'm not sure why that's the default styling, however; the breadcrumbs won't ever show up on the page with that styling the way it is.